frontend/src/components/DashboardPage.js

```diff
@@ -149,6 +152,19 @@ const DashboardPage = ({
               </Col>
             </Row>
           </Card>
+          <div style={{ display: 'flex', marginBottom: '1em' }}></div>
+          <Card title="Libraries" {...CARD_PROPS}>
```

Comment:
I would swap this card with the indices one and add the library_type statistic breakdown.